ekf_lolo_auv/src/landmark_ml.cpp

```diff
@@ -0,0 +1,86 @@
+#include "landmark_ml/landmark_ml.hpp"
+
+
+LandmarkML::LandmarkML(const boost::numeric::ublas::vector<int> &landmark_pos){
+    landmark_id_ = landmark_pos(0);
+    landmark_pos_ = boost::numeric::ublas::vector<int>(3);
+    landmark_pos_(0) = landmark_pos(1);
+    landmark_pos_(1) = landmark_pos(2);
+    landmark_pos_(2) = landmark_pos(3);
+}
+
+void LandmarkML::computeH(const boost::numeric::ublas::vector<double> &mu_hat,
+                          const tf::Vector3 lm_odom){
+
+    using namespace std;
+    H_ = boost::numeric::ublas::identity_matrix<double>(2,6);
+
+    H_(0,0) = cos(mu_hat(3))*sin(mu_hat(5)) - cos(mu_hat(5))*sin(mu_hat(4))*sin(mu_hat(3));
```

Comment:
Holy #"?¤=?#¤, I'm gonna trust you on this one...:)

Reply:
It's something matlab pucked and I copy pasted. But it works!